tests/helpers.jsx

```diff
@@ -172,6 +194,7 @@ module.exports = {
       },
       UserStore: {
```

Comment:
if you design your components in such a way that they simply receive data through props rather than fetching directly from the store, you wouldn't have to stub out stores. you would simply test your isolated components by passing in props.